Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -41,6 +41,18 @@ void NSStringForceinclude() {
 
 @implementation NSString (UIKitAdditions)
 
+static CTTextAlignment __UITextAlignmentToCTTextAlignment(UITextAlignment alignment) {
```

Comment:
Double checked - the values between CT/UI are different, and we can't change the values of NS/UI because our internal test app (and presumably others) are depending on those values staying the same #ByDesign